docs/versioning.md

```diff
@@ -1,8 +1,8 @@
-# Versioning
+# MFTF versioning
```

Comment:
Change to `MFTF versioning schema` to be consistent with Magento documentation in terminology.

Reply:
Fixed.